clients/python/exp_client/setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+setup(
+   name='exp_client',
```

Comment:
@pratikmishra356 python follows kebab case for package names
https://pypi.org/search/?q=django&o=